crates/fluvio-controlplane-metadata/src/mirror/status.rs

```diff
@@ -8,8 +8,9 @@ use fluvio_protocol::{Encoder, Decoder};
     serde(rename_all = "camelCase")
 )]
 pub struct MirrorStatus {
```

Comment:
is there reason why this is public?

Reply:
It's been used in other crates. For example, we're using it on SPU crate now to send to SC.